app/main.py

```diff
@@ -1,6 +1,25 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.people.cinema_staff import Cleaner
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str
+) -> None:
+
+    cinema_hall = CinemaHall(hall_number)
+    cleaning_staff = Cleaner(cleaner)
+
+    for customer in customers:
+        customer = Customer(customer["name"], customer["food"])
+        CinemaBar.sell_product(customer, customer.food)
+
+    cinema_hall.movie_session(
+        movie,
+        [Customer(name=i["name"], food=i["food"])
+         for i in customers],
+        cleaning_staff)
```

Comment:
```suggestion
    cinema_hall.movie_session(
        movie,
        [
            Customer(name=i["name"], food=i["food"])
            for i in customers
        ],
        cleaning_staff
    )
```

Try like that, it will make your code more readable. Also the last bracket should be placed like the first one (starts with a text or have a line break between the text), like I did in the suggestion